ReleaseNotes.md

```diff
@@ -3,7 +3,7 @@
 ## unreleased changes
 * [Issue #923](https://github.com/AmpersandTarski/Ampersand/issues/923) Lexer error message had disappeared. Now they show again.
 * [Issue #925](https://github.com/AmpersandTarski/Ampersand/issues/925) Add warning when a script contains `BOX <ROWSNH>`, for this is deprecated and could result in an error when a prototype is being generated.
-* [Issue #578](https://github.com/AmpersandTarski/Ampersand/issues/578) Implement meta model for navigation menu
```

Comment:
Why was the line about Issue#578 removed? It should be put back in, I guess.

Reply:
Sorry. My mistake. I reinserted it again.